build.gradle

```diff
@@ -259,6 +265,25 @@ dependencies {
     rewrite("org.openrewrite.recipe:rewrite-migrate-java")
 }
 
+spotbugs {
+    toolVersion = '4.8.0'
+    ignoreFailures = true
+    showStackTraces = true
+    showProgress = false
+}
+
+spotbugsMain {
+    reports {
+        html {
+            required = true
+            outputLocation = file("build/reports/spotbugs")
```

Comment:
Does spotbugs also write into `spotbugs` as indicated in `.gitignore`? If not, the whole `.gitignore` changes can be reverted.